wp-graphql-yoast-seo.php

```diff
@@ -19,349 +19,380 @@
 use WPGraphQL\AppContext;
 use WPGraphQL\Data\DataSource;
 
-function get_og_image($images)
-{
+add_action('graphql_init', function() {
+	$core_dependencies = [
+		'WPGraphQL plugin' => class_exists( 'WPGraphQL' ),
+		'Yoast SEO' => is_plugin_active('wordpress-seo/wp-seo.php')
+	];
+
+	$missing_dependencies = array_keys( array_diff( $core_dependencies, array_filter( $core_dependencies ) ) );
+	$display_admin_notice = static function () use ( $missing_dependencies ) {
+		?>
+		<div class="notice notice-error">
+			<p><?php esc_html_e( 'The WPGraphQL Offset Pagination plugin can\'t be loaded because these dependencies are missing:', 'wp-graphql-yoast-seo' ); ?></p>
```

Comment:
```suggestion
			<p><?php esc_html_e( 'The WPGraphQL Yoast SEO plugin can\'t be loaded because these dependencies are missing:', 'wp-graphql-yoast-seo' ); ?></p>
```